### ISSUE TITLE
explode if you bell too much

### ISSUE THREAD

**Issue by cgsdev0** [kuviman/salmoning#4]

**Comment by cgsdev0** [comment]
@kuviman any update on this one?

**Comment by kuviman** [comment]
I thought you were on it @cgsdev0

**Comment by cgsdev0** [comment]
@Nertsal can you take this one?

**Comment by kuviman** [comment]
I would take it if I was @Nertsal 100%

**Comment by cgsdev0** [comment]
can you see if @vesdev wants to make the sound effect @Nertsal

**Comment by cgsdev0** [comment]
and @ArcRincs could do the art too

**Comment by Nertsal** [comment]
i think @kuviman is the professional on explosions given how many biters he's exploded recently

are we playing salmoning in some post d2jam event or what's going on?

**Comment by pgorley** [comment]
I'm glad I subscribed to this repo.

🍿

**Comment by cgsdev0** [comment]
any update on this @pgorley ?

**Comment by pgorley** [comment]
yep no problem i've got it allllllllllllll under control 😄  :shipit:  🚀
